mods/ts/rules/trees.yaml

```diff
@@ -113,3 +125,78 @@ VEINHOLE:
 		Palette: player
 	WithSpriteBody:
 	AutoSelectionSize:
+
+FONA01:
+	Inherits: ^Tree
+	Tooltip:
+		Name: Fona
```

Comment:
Well, with Westwood you never know. They might just have mixed them up, in addition to the spelling. Or it's meant to indicate that these 'plants' are more like immobile aliens of sorts.